deploy/deploy_research_environment.yml

```diff
@@ -25,6 +25,71 @@
         regexp: ClientAliveCountMax
         line: ClientAliveCountMax=384
 
+    - name: additional symlinks for odbc.ini, odbcinst.ini, freetds, cups to shared env
+      file:
+        state: link
+        src: "{{ item.src }}"
+        dest: "{{ item.dest }}"
+      with_items: "{{ additional_symlinks }}"
+
+    - name: Add additional software to all Research servers
+      yum:
+         name: "{{ item }}" 
+         state: present
+      with_items:
+         - htop
+         - cups
+         - hplip
+         - gutenprint
+         - foomatic
+         - udev
+         - firefox
+         - system-config-printer
+         - postgresql-odbc
+ 
+    - name: Add Postgresl repository on all Research servers
+      yum: 
+        name: "{{ eod_postgres_repository_url }}" 
```

Comment:
Let's just call this "postgres_repository_url", the same as in the postgres-client script.  Or better yet, maybe just assign the postgres-client role to all research environment servers, and make sure that role contains all the packages needed, rather than adding these separate tasks.